packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts

```diff
@@ -300,12 +337,12 @@ export function createMikrORMEntity<
   /**
    * Processing schema fields
    */
-  Object.keys(entity.schema).forEach((property) => {
-    const field = entity.schema[property].parse(property)
+  Object.keys(schema).forEach((property) => {
```

Comment:
NIT: You can use `.entries` so you don't ned the `schema[property]`

Reply:
Sure thing!